Archipelago.MultiClient.Net/Helpers/LocationCheckHelper.cs

```diff
@@ -172,16 +175,41 @@ public void CompleteLocationChecksAsync(Action<bool> onComplete, params long[] i
         /// <exception cref="T:Archipelago.MultiClient.Net.Exceptions.ArchipelagoSocketClosedException">
         ///     The websocket connection is not alive.
         /// </exception>
-        public void ScoutLocationsAsync(Action<LocationInfoPacket> callback = null, params long[] ids)
+        public void ScoutLocationsAsync(Action<LocationInfoPacket> callback = null, bool createAsHint = false, params long[] ids)

```

Comment:
I prefer the way by adding an additional parameter, i just dont see why you added an overload? the original behaviour is already preserved by the default value of false right?

With that i woud like to see the overload removed

Reply:
Never mind that, i just tried it locally, it seems optional arguments and params array aren't as flexable as i expected them to be